Dockerfile.rocm

```diff
@@ -1,8 +1,32 @@
-FROM rocm/pytorch:rocm5.7_ubuntu22.04_py3.10_pytorch_2.0.1
+# default base image
+ARG BASE_IMAGE="rocm/pytorch:rocm5.7_ubuntu22.04_py3.10_pytorch_2.0.1"
```

Comment:
Should this be defined twice?   

Also any particular reason we aren't using ROCm 6.0 as the default?

Reply:
yes, it has to be as anything before FROM has no effect after FROM.